packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -384,6 +384,14 @@ export interface ComboBoxProps<ItemType>
    * Provide the text that is displayed when the control is in warning state
    */
   warnText?: ReactNode;
+
+  /**
+   * Props to be passed directly to the underlying <input> element.
+   * Useful for customizing input behavior, such as setting maxLength, placeholder and other native input attributes.
+   *
+   * Note: These props are merged with internal input props. Custom values will override defaults when applicable.
+   */
+  inputProp?: InputHTMLAttributes<HTMLInputElement>;
```

Comment:
I think plurality makes more sense since it accepts multiple attributes.

```suggestion
  inputProps?: InputHTMLAttributes<HTMLInputElement>;
```

Reply:
Updated